### ISSUE TITLE
Just curious if you will be developing this any more?

### ISSUE THREAD

**Issue by BertilJ** [arjanmels/RoboMowRC#2]
I'd be happy to support with testing if that is the case.

I've got a RS625 (2017 model).

**Comment by xjustbmw** [comment]
i am also interested

**Comment by geelenbert** [comment]
I’m also interested in this project. 

Would love to integrate my robomow with home assistant

**Comment by ChristofferG** [comment]
I am interested as well! And have ordered the board and hope to be able to help a bit on the development.

**Comment by Sprocket02** [comment]
I am interested as well!

**Comment by xjustbmw** [comment]
no updates on this ?

**Comment by joostwestra** [comment]
I would not mind spending some time on it. Does the current version compile for any of you? (I get multiple problems at the moment)

**Comment by joostwestra** [comment]
I have to code compiled, deployed, connected and authenticated to my RoboMow RX. I am receiving the relevant updates over mqtt. 
Does anyone know if control is already implemented? And if yes how to use it?

**Comment by geelenbert** [comment]
I want to get going on this also this summer. I've got an RS630 in mind i want tot connect it to.

What hardware/setup did you use @joostwestra ?

**Comment by joostwestra** [comment]
Currently using a full esp-wrover-kit, which is a bit of overkill.
But you do need an esp32 with psram, otherwise it currently crashes (I tested that)

**Comment by xjustbmw** [comment]
Care to share the code with us?

**Comment by promy** [comment]
Quite some time ago i bought a TTGO T-Beam board board just to run this ;-) Its compiling, connecting and publishing to mqtt, but... (basically same questions as joostwestra ;-)
- can anyone tell me how to send commands to it? or maybe made a command page on the esp?
- would anyone already have made a status page (on the esp?)
- it seems that after every reboot all the settings are missing (serial, mqtt server), is this normal?

**Comment by BSD31** [comment]
Hello.
Is this project still under development?
I am very interested.
Thank you

Bonjour.
Ce projet est-il encore en cours de développement ?
Je suis très intéressé.
Merci

**Comment by joostwestra** [comment]
I does not appear to be maintained anymore at all. I personally would not mind patches some small glitches. 
But as far as I can tel the project current does not implement sending any commands to the robot at all.

**Comment by TheCutter** [comment]
I'm working on my own MQTT-Robomow interface for some time now. I also use an ESP32 to connect to my RS615u and send him commands. Can't share any code yet, but will keep you updated!

**Comment by geelenbert** [comment]
@TheCutter thats interesting!
What hardware platform are you using ?
I'm interested in testing if you like.

**Comment by TheCutter** [comment]
I'm using an ESP-WROOM-32 Devboard. My plan is to start mowing from my Node-Red Installation via MQTT, based on time, weather and other properties.

**Comment by promy** [comment]
I would also be interested in this! would it be possible to get status updates as well? For logging how when it leaves and returns, of get a push message when there is a problem (stuck, something in the mowing deck,...)?

**Comment by TheCutter** [comment]
Yes sure. Current State is also planed. But it will be a polling update every x minutes with MQTT. 
At least that's my plan. :)

**Comment by TheCutter** [comment]
I created a new repository to discuss further ideas over here:
https://github.com/TheCutter/RoboMow2MQTT/issues/1

**Comment by MortenVinding** [comment]
I stumbled upon this:
https://www.skyynet.de/mowgli_connect.php

But it is all in german, so I made my own repo about it and also made a really dirty script to translate the german in the (binary) file.
https://github.com/MortenVinding/Robomow-ESP32

Let me know if it's useful and if I should change anything.

**Comment by ChristofferG** [comment]
Cool, I will check what firmware my Robomow is running and hopefully give this a try!
